src/olympia/editors/tests/test_forms.py

```diff
@@ -45,47 +47,71 @@ def test_other_statuses(self):
             if status in statuses:
                 return
             else:
-                label = self.set_status(status)['prelim']['label']
+                status = self.set_statuses(addon_status=status,
+                                           file_status=amo.STATUS_UNREVIEWED)
+                label = status['prelim']['label']
                 assert force_unicode(label) == 'Grant preliminary review'
 
     def test_nominated_unlisted_addon_no_prelim(self):
         self.addon.update(is_listed=False)
-        actions = self.set_status(amo.STATUS_NOMINATED)
+        actions = self.set_statuses(addon_status=amo.STATUS_NOMINATED,
+                                    file_status=amo.STATUS_UNREVIEWED)
         assert 'prelim' not in actions
         assert actions['public']['label'] == 'Grant full review'
 
     def test_reject(self):
-        reject = self.set_status(amo.STATUS_UNREVIEWED)['reject']['details']
+        status = self.set_statuses(addon_status=amo.STATUS_UNREVIEWED,
+                                   file_status=amo.STATUS_UNREVIEWED)
+        reject = status['reject']['details']
         assert force_unicode(reject).startswith('This will reject the add-on')
 
     def test_reject_lite(self):
-        reject = self.set_status(amo.STATUS_LITE)['reject']['details']
+        status = self.set_statuses(addon_status=amo.STATUS_LITE,
+                                   file_status=amo.STATUS_UNREVIEWED)
+        reject = status['reject']['details']
         assert force_unicode(reject).startswith('This will reject the files')
 
     def test_not_public(self):
-        # If the file is unreviewed then there is no option to reject,
-        # so the length of the actions is one shorter
-        assert len(self.set_status(amo.STATUS_UNREVIEWED)) == 5
```

Comment:
The old comment did not make any sense at all. I changed it to what the test is actually testing.